develop/develop-images/build_enhancements.md

```diff
@@ -1,50 +1,61 @@
 ---
 title: Build Enhancements for Docker
```

Comment:
As the toc has been updated to `Build images with BuildKit`, should we also specify BuildKit in the title?